src/desktop/components/main_layout/stylesheets/slick-css.styl

```diff
@@ -0,0 +1,273 @@
+@charset 'UTF-8';
+.slick-dots,
```

Comment:
Has this been tested? I'm not sure http://stylus-lang.com/  is 100% compatible with CSS (as  in, you can just copy / paste in) but it could be! 

Can see the slider in 
- artist/id header 
- article pages that have multiple images 
- artwork/id main carousel 
- Recently viewed slider at the bottom of new pages 
- maybe a few others... 

@eessex - Do you know of any other locations?

Reply:
ah good point. I took a look at the recently viewed carousel and it was working correctly with this version, but I'll check those other locations.

we could also just add an `@css` wrapping everything to be sure http://stylus-lang.com/docs/literal.html

Reply:
I think the `@css` helper would work great 👍

Reply:
updated! and I checked all of the pages you mentioned above, looking good to me. Will merge assuming tests pass